NitroxClient/GameLogic/PlayerGameObjectManager.cs

```diff
@@ -17,6 +20,30 @@ public void UpdatePlayerPosition(string playerId, Vector3 position, Quaternion r
             MovementHelper.MoveGameObject(player, position, rotation, PLAYER_TRANSFORM_SMOOTH_PERIOD);
         }
 
+        public void UpdateAnimation(string playerId, AnimChangeType type, AnimChangeState state)
+        {
+            GameObject player = GetPlayerGameObject(playerId);
+            GameObject playerView = player.transform.Find("player_view").gameObject;
+            AnimationController controller = playerView.GetComponent<AnimationController>();
+
+            bool value;
+            if (state == AnimChangeState.Off)
```

Comment:
In the future, this is going to change a lot. I originally had this, and depending on how the other animations turn out, I may either have to keep this and modify a lot or use this.